meta-mender-core/recipes-mender/mender-connect/mender-connect.inc

```diff
@@ -21,6 +21,9 @@ inherit systemd
 GO_IMPORT = "github.com/mendersoftware/mender-connect"
 
 do_compile() {
+    # Remove file system paths from binary and use buildmode PIE to avoid text relocations
+    export GOFLAGS="-trimpath -buildmode=pie"
```

Comment:
`-trimpath` we should not apply it here. Let's rely on the Makefile (and it will be fixed only for the next releases)

Reply:
And with regards to `-buildmode=pie` I believe it is not working. This will only apply to the Go code and not the C code. For C code you need the equivalent gcc flag in `CGO_LDFLAGS`. Additionally, we should not do this hack-ish export but just amend to the yocto variable (out of the `do_compile`).